USAGE.md

```diff
@@ -0,0 +1,104 @@
+# Amazon Alexa Smart Home Skill for openHAB 2
+
+Amazon certified openHAB2 as a [smarthome skill for Alexa](https://www.amazon.com/openHAB-Foundation/dp/B01MTY7Z5L).
+This page describes how to use the skill certified by Amazon.
+This skill allows you to connect your openHAB setup through the [myopenHAB.org](http://myopenHAB.org) cloud service to Amazon Alexa.
+(See [README.md](https://github.com/openhab/openhab-alexa/blob/master/README.md) for other setup options and development information.)
+
+With this skill you can control items that respond to on/off and numeric commands.
+The Skill as well has limited support for thermostats.
+The skill is supported for English (U.S.), English (U.K.), and German languages. 
+
+## General Configuration Instructions
+
+### Requirements
+
+* [openHAB Cloud Connector](http://docs.openhab.org/addons/io/openhabcloud/readme.html) configured using myopenHAB.org
+* Amazon account
+* Amazon Echo, Amazon Echo Dot or compatible Alexa speaker or solution (e.g. [Reverb](https://reverb.ai/))
+
+### 1. Item Configuration
+
+You have to tag your items to make them useable by Alexa.
```

Comment:
I know its mentioned down the page a bit, but I think a link to the homekit binding syntax it very important to have before we show examples.  So I would change this to:

```Items are exposed to Alexa through the use of tags which follow the [Homitkit](http://docs.openhab.org/addons/io/homekit/readme.html) binding tagging syntax.  Pleas see the [HomeKit item configuration](http://docs.openhab.org/addons/io/homekit/readme.html#item-configuration)  page for information on how to tag items. ```

Reply:
OK